insights/specs/default.py

```diff
@@ -165,13 +165,13 @@ def is_ceph_monitor(broker):
 
     ceph_insights = simple_command("/usr/bin/ceph insights", deps=[is_ceph_monitor])
     ceph_log = simple_file("/var/log/ceph/ceph.log")
-    ceph_osd_dump = simple_command("/usr/bin/ceph osd dump -f json-pretty")
-    ceph_osd_df = simple_command("/usr/bin/ceph osd df -f json-pretty")
+    ceph_osd_dump = simple_command("/usr/bin/ceph osd dump -f json")
+    ceph_osd_df = simple_command("/usr/bin/ceph osd df -f json")
     ceph_osd_ec_profile_ls = simple_command("/usr/bin/ceph osd erasure-code-profile ls")
-    ceph_osd_ec_profile_get = foreach_execute(ceph_osd_ec_profile_ls, "/usr/bin/ceph osd erasure-code-profile get %s -f json-pretty")
+    ceph_osd_ec_profile_get = foreach_execute(ceph_osd_ec_profile_ls, "/usr/bin/ceph osd erasure-code-profile get %s -f json")
```

Comment:
@psachin you need to add an archive spec for this one.  Other than that it looks good.

Reply:
@bfahr Sure. Specs are now added in the same commit. Thanks.